microsetta_public_api/models/_exceptions.py

```diff
@@ -0,0 +1,6 @@
+class UnknownID(ValueError):
+    pass
+
+
+class DisjointError(KeyError):
```

Comment:
If so, what's a DisjointError?

Reply:
You could envision passing a function two tables and it should error if they do not have the same keys.